Scripts/DCS-BIOS/test/mocking_compile/DCS_API_mocks.lua

```diff
@@ -0,0 +1,280 @@
+
+--- @func Gets a device based on the device ID as specified in devices.lua
+--- @param device_id integer
+--- @return CockpitDevice
+function GetDevice(device_id)
+	return CockpitDevice
+end
+
+--- @func Gets the draw value for a certain animation
+--- @param draw_argument_id integer
+--- @return number
+function LoGetAircraftDrawArgumentValue(draw_argument_id)
+	return 0
+end
+
+--- Contains Postions and Altitude
+--- @class LatLongAlt
+--- @diagnostic disable-next-line: duplicate-doc-field
+--- @field Lat number
+--- @diagnostic disable-next-line: duplicate-doc-field
+--- @field Long number
+--- @diagnostic disable-next-line: duplicate-doc-field
+--- @field Alt number Altitude in meters
+LatLongAlt = {}
+
+--- Contains Aircraft Data
+--- @class AicraftData
+--- @diagnostic disable-next-line: duplicate-doc-field
+--- @field Name string
+--- @diagnostic disable-next-line: duplicate-doc-field
+--- @field Heading number
+--- @diagnostic disable-next-line: duplicate-doc-field
+--- @field LatLongAlt LatLongAlt
+AicraftData = { Name = "Test", Heading = 0, LatLongAlt({ Lat = 0, Long = 0, Alt = 0 }) }
+
+--- @func Returns SelfData that holds information about the aircraft
+--- @return AicraftData
+function LoGetSelfData()
+	return AicraftData
+end
+
+--- @func Returns the simulation time
+--- @return number
+function LoGetModelTime()
+	return 0
+end
+
+--- @func Returns the mission start time
+--- @return number
+function LoGetMissionStartTime()
+	return 0
+end
+
+--- @func Returns whether own ships/aircraft data can be exported.
+--- @return boolean
+function LoIsOwnshipExportAllowed()
+	return true
+end
+
+--- @func Returns pilot's name
+--- @return string
+function LoGetPilotName()
+	return "Neil Armstrong"
+end
+
+--- @func Returns indicated airspeed
+--- @return number
+function LoGetIndicatedAirSpeed()
+	return 0
+end
+
+--- Contains acceleration info for all dimensions
+--- @class AccelerationUnit
+--- @diagnostic disable-next-line: duplicate-doc-field
+--- @field x number
+--- @diagnostic disable-next-line: duplicate-doc-field
+--- @field y number
+--- @diagnostic disable-next-line: duplicate-doc-field
+--- @field z number
+AccelerationUnit = { x = 0, y = 0, z = 0 }
+
+--- @func Returns G Load
+--- @return AccelerationUnit
+function LoGetAccelerationUnits()
+	return AccelerationUnit
+end
+
+--- @func Returns ADI pitch, band, yaw
+--- @return number
+function LoGetADIPitchBankYaw()
+	return unpack({ 0, 0, 0 })
+end
+
+--- @class CounterMeasures
+--- @diagnostic disable-next-line: duplicate-doc-field
+--- @field chaff number
+--- @diagnostic disable-next-line: duplicate-doc-field
+--- @field flare number
+CounterMeasures = { chaff = 0, flare = 0 }
+
+--- @func Returns information about countermeasures
+--- @return CounterMeasures
+function LoGetSnares()
+	return CounterMeasures
+end
+
+--- @func Returns a list of strings for a cockpit indicator (screen)
+--- @return table<string>
```

Comment:
For future reference, you can specify this as `string[]`

There are others like this in this file as well

Reply:
OK

Reply:
done